en/tutorials-and-examples/blog/part-two.rst

```diff
@@ -261,7 +262,8 @@ ArticlesController::
 
         public function index()
         {
-            $this->set('articles', $this->Articles->find('all'));
+            $articles = $this->set('articles', $this->Articles->find('all'));
```

Comment:
@miguelarcher you either didn't push your commit; either still not corrected the code example.